### PR TITLE
more designer pages translated to ru

### PR DESCRIPTION
**Pull request by yak32**
keyboard shortcuts page is translated to ru

### CODE REVIEW

**Review by yaustar — APPROVED** [review]
LGTM

File: content/ru/user-manual/packs/components/camera.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ The Camera component can be enabled or disabled using the toggle in the top righ
 
 ## Properties
 
-| Property        | Description |
+| Property        | Описание |
 |-----------------|-------------|
 | Clear Buffers   | Controls how the camera's render target is cleared before rendering each frame. The camera maintains two types of buffer: the color buffer which stores the visible image and a depth buffer that stores information about the distance of a rendered pixel in relation to the position of the camera. The buffers can be cleared independently: <ul><li>Color: If selected, the camera will explicitly clear its render target to the chosen clear color before rendering the scene.</li><li>Depth: If selected, the camera will explicitly clear the depth buffer of its render target before rendering the scene.</li></ul> |
-| Clear Color     | The color used to clear the camera's render target.  This property is only displayed if 'Clear Color Buffer' is selected. |
+| Цвет очистки экрана     | The color used to clear the camera's render target.  This property is only displayed if 'Clear Color Buffer' is selected. |
```
Comment (mvaligursky) [inline content/ru/user-manual/packs/components/camera.md:18]: should the property name be translated?

File: content/ru/user-manual/packs/components/element.md [hunk 2]
```diff
@@ -37,30 +37,30 @@ The text element renders a string of text using a [font asset][4].
...
 ## Image Component Properties
 
-| Property | Description |
+| Property | Описание |
 |----------|-------------|
 | Rect     | Define the area of the texture asset to display. |
 | Mask     | Switch Image Element into a mask. Masks do not render into the scene, but instead limit child elements to only be rendered where this element is rendered. |
 | Texture  | The texture asset displayed. |
-| Color    | The color to tint the element. |
+| Цвет    | The color to tint the element. |
```
Comment (mvaligursky) [inline content/ru/user-manual/packs/components/element.md:50]: here as well, property name should stay?

Comment (yak32) [reply content/ru/user-manual/packs/components/element.md:50]: yes, I will update it with the next PR

**Pull request merged**